code/src/ProjectTemplates/UWP/CS/Ts4UwpCsTemplate/Ts4UwpCsTemplate.de-DE.vstemplate

```diff
@@ -23,8 +23,7 @@
 		<CustomParameters>
 			<CustomParameter Name="$ts.platform$" Value="Uwp"/>
 			<CustomParameter Name="$ts.language$" Value="C#"/>
-			<CustomParameter Name="$ts.category$" Value="TemplateStudio"/>
-			<CustomParameter Name="$ts.version$" Value="5.0"/>
+			<CustomParameter Name="$ts.category$" Value="TemplateStudio"/>S
```

Comment:
Remove this 'S' at the end.

Reply:
done - Although I'm a tad concerned that the build system didn't pick this up as the file is invalid. :/